learn/how-tos/changing-app-logo.md

```diff
@@ -14,7 +14,7 @@ App Logo is used in multiple places, for example:
 
 By default WaveMaker logo is used as app logo. You can change it by replacing the logo file.
 
-1. From the Main Menu, under **Import -> Resource** option locate the **logo** under the images folder. 
```

Comment:
There are two ways to change the app logo. One, from the header instead of "main menu", and the other way is to change it from the file explorer.